jwst/photom/photom.py

```diff
@@ -665,17 +665,25 @@ def photom_io(self, tabdata, order=None):
                 else:
                     srctype = self.input.meta.target.source_type
                 if srctype is None or srctype.upper() != 'POINT':
-                    log.debug('Converting conversion factor from flux '
-                              'to surface brightness')
-                    conversion /= slit.meta.photometry.pixelarea_steradians
+                    if slit.meta.photometry.pixelarea_steradians is None:
```

Comment:
For the case that failed in the regression tests, was the problem due to the pixel area missing from the reference file? Or was the loading of it missed somewhere else in the code?

Reply:
In function `save_area_info`, no value was being assigned to the `pixel_steradians` attribute if the relevant value was not present in the area reference file.  I think that's appropriate, but I needed to check for that case when `pixel_steradians` was used.